src/webgl/text.js

```diff
@@ -167,9 +169,13 @@ class FontInfo {
      * calculates rendering info for a glyph, including the curve information,
      * row & column stripes compiled into textures.
      */
-  getGlyphInfo (glyph) {
+  getGlyphInfo (glyph, variableValues) {
```

Comment:
Right now nothing passes in `variableValues` to this function, can we also edit the spots that call `getGlyphInfo` to pass along this information?